### ISSUE TITLE
[MAS1.3.1][Screen Reader-Add other service] After error, the voiceover announced incorrect label of the edit fields.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1911]
**Actual Result:**
The voiceover announces the correct label before the any of the field has error. After error it announces the same error message to every edit field.
Voiceover announces "Service Name*, Invalid required data unselected" before the error "The field cannot be empty", after the error it reads same error message at every edit field.

**Expected Result:**
The voiceover should reads the error only on the edit field where the error exist and on the other edit field the labels should be announced which are already defined for them.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Service" and click on "+".
4. Select "Add other service" from the "Menu".
5. Enter any data in "Service Name" and delete them.
6. An error "The field cannot be empty" appears, ignore the error and press "Tab".
7. Observe the issue.

**User Impact:**
Users who depends on the Screen reader will get confused if same error is on the every controls.

**Note:**
Same issue observed if user "edit and delete" any required field.
Same issue observed in edit fields present at "Add a LUIS app manually" pane.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment:**
[Incorrect label after error.zip](https://github.com/microsoft/BotFramework-Emulator/files/3691026/Incorrect.label.after.error.zip)

**Comment by tonyanziano** [comment]
Fixed by #1970

**Comment by ShikhaMishra11** [comment]
Issue partially fixed, now the voiceover reads the label of edit fields correctly but still it doesn't read the label of the edit field with error. Please find the attachment below:
[Label of edit field having error.zip](https://github.com/microsoft/BotFramework-Emulator/files/3827226/Label.of.edit.field.having.error.zip)

Environment:
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.6.0-89098)

**Comment by tonyanziano** [comment]
@denscollo The issue was only partially fixed. Could you please take a look at the attachment and see what needs to be done to resolve it?

**Comment by denscollo** [comment]
@tonyanziano Sure, we'll review the attachment and find a solution. Thanks!

**Comment by ShikhaMishra11** [comment]
Fixed, Verified on MAC build 10.15 Beta(19A573a)
